templates/docugami-kg-rag/docugami_kg_rag/config.py

```diff
@@ -0,0 +1,94 @@
+import os
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List
+
+from langchain.cache import SQLiteCache
+from langchain.chat_models import ChatOpenAI
+from langchain.embeddings import OpenAIEmbeddings
+from langchain.globals import set_llm_cache
+from langchain.storage.in_memory import InMemoryStore
+
+LLM = ChatOpenAI(temperature=0, model="gpt-4-1106-preview")
+
+EMBEDDINGS = OpenAIEmbeddings(model="text-embedding-ada-002")
+EMBEDDINGS_DIMENSIONS = 1536  # known size of text-embedding-ada-002 embeddings
+
+DOCUGAMI_API_KEY = os.environ.get("DOCUGAMI_API_KEY")
+if not DOCUGAMI_API_KEY:
+    raise Exception("Please set the DOCUGAMI_API_KEY environment variable")
+
+PINECONE_INDEX = os.environ.get("PINECONE_INDEX")
```

Comment:
maybe rename env var to `PINECONE_INDEX_PREFIX` to make clear a docugami docset id will be appended to the end

Reply:
This env var has been removed now, since we are using Chroma (and no prefix is needed). Hence closing this comment, thanks.